packages/components/src/utils/getAtlantisConfig.ts

```diff
@@ -0,0 +1,7 @@
+// For retheme purposes only.
+// This will be removed once the retheme is out of the experiment.
+export function getAtlantisConfig(): typeof window.ATLANTIS_CONFIG {
+  if (typeof window !== "undefined") {
+    return window.ATLANTIS_CONFIG;
+  }
+}
```

Comment:
```suggestion
// For retheme purposes only.
// This will be removed once the retheme is out of the experiment.
export function getAtlantisConfig() {
  const rethemePresent =
    getComputedStyle(document.body).getPropertyValue("--jobber-retheme") ===
    "true";

  return {
    JOBBER_RETHEME: rethemePresent,
  };
}
```

As discussed, move the condition as a CSS var that we read in JS to determine if we're rethemeing.